parity/rpc_apis.rs

```diff
@@ -336,6 +340,14 @@ impl FullDependencies {
 						handler.extend_with(client.to_delegate());
 					}
 				}
+				Api::TransactionsPool => {
+					if !for_generic_pubsub {
+						let receiver = self.miner.get_tx_pool_receiver();
+						let client = TransactionsPoolClient::new(self.executor.clone());
+						client.run(receiver);
```

Comment:
Why not pass it in `new`? Now initialisation requires those two methods to be called precisely in order.